### ISSUE TITLE
Adding `GridFieldOrderableRows` works but can't move items between pages

### ISSUE THREAD

**Issue by sinan-evanshunt** [silverstripe/silverstripe-lumberjack#101]
To be fair, not sure if this is a lumberjack issue or gridfield or COVID. But here is what I have done to allow lumberjack gridfield items be sortable:

```
use SilverStripe\Lumberjack\Forms\GridFieldConfig_Lumberjack;
use Symbiote\GridFieldExtensions\GridFieldOrderableRows;

public function getLumberjackGridFieldConfig() {
        $lumberjackGridfieldConfig = GridFieldConfig_Lumberjack::create();
        $lumberjackGridfieldConfig->addComponent(new GridFieldOrderableRows('Sort'));
        return $lumberjackGridfieldConfig;
    }
```

Now that worked great, I can manually sort the items by dragging and dropping them. But if I wanted to move one item from one page to another by dragging it to the pagination arrows, I get this JS console error:

<img width="561" alt="image" src="https://user-images.githubusercontent.com/26654500/101067235-892ec200-3554-11eb-9f86-609e2bab8905.png">

`
Uncaught TypeError: this.helper is null       vendor.js:1:2037761
    _clear http://localhost/resources/vendor/silverstripe/admin/client/dist/js/vendor.js?m=1591934438:1
    ./thirdparty/jquery-ui/jquery-ui.js/</</e.widget/</r[t]</< http://localhost/resources/vendor/silverstripe/admin/client/dist/js/vendor.js?m=1591934438:1
    _mouseStop http://localhost/resources/vendor/silverstripe/admin/client/dist/js/vendor.js?m=1591934438:1
    ./thirdparty/jquery-ui/jquery-ui.js/</</e.widget/</r[t]</< http://localhost/resources/vendor/silverstripe/admin/client/dist/js/vendor.js?m=1591934438:1
    _mouseUp http://localhost/resources/vendor/silverstripe/admin/client/dist/js/vendor.js?m=1591934438:1
    ./thirdparty/jquery-ui/jquery-ui.js/</</e.widget/</r[t]</< http://localhost/resources/vendor/silverstripe/admin/client/dist/js/vendor.js?m=1591934438:1
    _mouseUpDelegate http://localhost/resources/vendor/silverstripe/admin/client/dist/js/vendor.js?m=1591934438:1
    dispatch http://localhost/resources/vendor/silverstripe/admin/client/dist/js/vendor.js?m=1591934438 line 1 > eval:2726
    eventHandle http://localhost/resources/vendor/silverstripe/admin/client/dist/js/vendor.js?m=1591934438 line 1 > eval:2384
    add http://localhost/resources/vendor/silverstripe/admin/client/dist/js/vendor.js?m=1591934438 line 1 > eval:2421
    on http://localhost/resources/vendor/silverstripe/admin/client/dist/js/vendor.js?m=1591934438 line 1 > eval:3135
    each http://localhost/resources/vendor/silverstripe/admin/client/dist/js/vendor.js?m=1591934438 line 1 > eval:487
    each http://localhost/resources/vendor/silverstripe/admin/client/dist/js/vendor.js?m=1591934438 line 1 > eval:170
    on http://localhost/resources/vendor/silverstripe/admin/client/dist/js/vendor.js?m=1591934438 line 1 > eval:3134
    name http://localhost/resources/vendor/silverstripe/admin/client/dist/js/vendor.js?m=1591934438 line 1 > eval:3232
    js http://localhost/resources/vendor/silverstripe/admin/client/dist/js/vendor.js?m=1591934438:1
    js http://localhost/resources/vendor/silverstripe/admin/client/dist/js/vendor.js?m=1591934438:1
    js http://localhost/resources/vendor/silverstripe/admin/client/dist/js/vendor.js?m=1591934438:1
    t http://localhost/resources/vendor/silverstripe/admin/client/dist/js/vendor.js?m=1591934438:1
    js http://localhost/resources/vendor/silverstripe/admin/client/dist/js/vendor.js?m=1591934438:1
    t http://localhost/resources/vendor/silverstripe/admin/client/dist/js/vendor.js?m=1591934438:1
    <anonymous> http://localhost/resources/vendor/silverstripe/admin/client/dist/js/vendor.js?m=1591934438:1
    <anonymous> http://localhost/resources/vendor/silverstripe/admin/client/dist/js/vendor.js?m=1591934438:1
​`

ANy insight or how could I fix this would be greatly appreciated.

**Comment by michalkleiner** [comment]
It's a known gridfield issue. The best workaround we use, so far, is providing configurable paginator with a config where the items can happen to be on the same page.

**Comment by sinan-evanshunt** [comment]
Yeah, I had this idea too if it was not possible to fix it easily. Thanks for the comment.

**Comment by gdelavil** [comment]
Hi all. I just wanted to check if there's been any changes related to this issue, or if the best approach is still to provide custom pagination? Just having to solve this on a project at the moment. Thanks!
